views/js/provider/group.js

```diff
@@ -55,17 +55,18 @@ define(['jquery', 'lodash', 'i18n', 'util/url', 'core/promise'], function ($, _,
                 config = _.defaults(config || {}, _defaults);
 
                 return new Promise(function(resolve, reject){
-
-                    $.ajax({
-                        url: urlUtil.route('addInstance', 'Groups', 'taoGroups'),
-                        type: 'post',
-                        data: config,
-                        dataType: 'json'
-                    })
-                        .done(function (group) {
+                        coreRequest({
+                            url: urlUtil.route('addInstance', 'Groups', 'taoGroups'),
+                            method: 'POST',
+                            type: 'post',
+                            data: config,
+                            dataType: 'json',
+                            noToken: false
+                        })
```

Comment:
You can safely remove the parameters `type` (is not checked anyway) and `noToken` (default is false in this function).